polaris/evaluate/metrics/docking_metrics.py

```diff
@@ -0,0 +1,67 @@
+# This script includes docking related evaluation metrics.
+
+from typing import Union, List
+
+import numpy as np
+from rdkit.Chem.rdMolAlign import CalcRMS
+import datamol as dm
+
+
+## The following rmsd implementation are modified based on ppsebusters https://github.com/maabuu/posebusters/blob/main/posebusters/modules/rmsd.py
+
```

Comment:
**Question:** Is this still accurate? Maybe we remove it? Or at least explain the difference to the original implementation?